vector/src/main/java/im/vector/fragments/VectorSettingsPreferencesFragment.java

```diff
@@ -207,6 +208,7 @@ public void onAccountInfoUpdate(MyUser myUser) {
     private EditTextPreference mSyncRequestDelayPreference;
     private PreferenceCategory mLabsCategory;
 
+
```

Comment:
Please do not commit useless change (like extra line here).
(except if they come from code formatting tool)